### DISCUSSION TITLE
Welcome to the WotL Texture Pack Discussions Page!

### DISCUSSION THREAD

**Discussion by Zodi-ark** [Zodi-ark/Final-Fantasy-Tactics-The-War-of-the-Lions-Texture-Pack#4] [Announcements]
##   Welcome to the People of Ivalice!

 _♈︎ ♉︎ ♊︎ ♋︎ ♌︎ ♍︎ ♎︎ ♏︎ ♐︎ ♑︎ ♒︎ ♓︎ ⛎︎_

  Hoping to use the discussions page as a place for the community to help build and refine the texture pack.
  From the humble, "press O to close" icon image to the chonky shop menus and annoying bugs...

  Don't be afraid to be apart of the project in any way.
  Even the smallest assets are appreciated.

  So...

  * Feel free to request to use your own artwork and contribute to the project.
  * Ask questions you’re wondering about (even the silliest ones).
  * Share ideas, bugs, critiques, and general sentiments.
  * Collaborate with other community members.
  * Encourage other artists to lend their talent.
  * Share the project with the public!

  Or if you just want to tell us how the project is coming along that is highly appreciated!

  If you even made it here, thank you.

**Comment by odevleo** [comment]
Hello mate, i'm using this pack in my ff t on the ppsspp emulator and looks great, the textures you made're very nice. Can you tell me how exactly are the config in you emulator? I'm having a little problem with fps drops in animation like special atacks, do you know how to solve this?

**Reply by Zodi-ark** [reply]
I copy-pasted all my graphic settings from my ppsspp ini file below.
If you're having big fps drops it might be a sign you have the setting ''save new textures'' accidentally enabled.
Also if you're not already using the latest PPSSPP version you should as it improves performance for texture replacement!

[Graphics]
EnableCardboardVR = False
CardboardScreenSize = 50
CardboardXShift = 0
CardboardYShift = 0
iShowStatusFlags = 0
GraphicsBackend = 2 (DIRECT3D11)
FailedGraphicsBackends = 
DisabledGraphicsBackends = 
VulkanDevice = 
D3D11Device = 
CameraDevice = 
AndroidFramerateMode = 1
UseGeometryShader = False
SkipBufferEffects = False
DisableRangeCulling = False
SoftwareRenderer = False
SoftwareRendererJit = True
HardwareTransform = True
SoftwareSkinning = True
TextureFiltering = 1
Smart2DTexFiltering = False
InternalResolution = 8
AndroidHwScale = 1
HighQualityDepth = 1
FrameSkip = 0
FrameSkipType = 0
AutoFrameSkip = False
StereoRendering = False
StereoToMonoShader = RedBlue
FrameRate = 0
FrameRate2 = -1
AnalogFrameRate = 240
UnthrottlingMode = CONTINUOUS
AnisotropyLevel = 4
MultiSampleLevel = 0
TextureBackoffCache = False
FullScreen = False
FullScreenMulti = False
BufferFiltering = 1
DisplayOffsetX = 0.500000
DisplayOffsetY = 0.500000
DisplayScale = 1.000000
DisplayIntegerScale = False
DisplayAspectRatio = 1.000000
DisplayStretch = False
DisplayCropTo16x9 = True
ImmersiveMode = True
SustainedPerformanceMode = False
IgnoreScreenInsets = True
ReplaceTextures = True
SaveNewTextures = False
IgnoreTextureFilenames = False
TexScalingLevel = 1
TexScalingType = 0
TexDeposterize = False
TexHardwareScaling = False
VSync = False
BloomHack = 0
SplineBezierQuality = 2
HardwareTessellation = False
TextureShader = Off
ShaderChainRequires60FPS = False
SkipGPUReadbackMode = 0
LogFrameDrops = False
InflightFrames = 3
RenderDuplicateFrames = False
MultiThreading = True
GpuLogProfiler = False
UberShaderVertex = True
UberShaderFragment = True

**Reply by odevleo** [reply]
I copied and pasted the settings this way but the problem still persists. And yes, the emulator has the latest version installed. But I think I understand the problem. The fps drop occurs when performing an action, using a potion, skill, I noticed that these animations normally have a half fps drop. The normal thing would be to drop from 60 to 30, but it's dropping from 30 to 15. If you know how to resolve this, I'd be grateful.

**Reply by odevleo** [reply]
I solved the problem. It turns out that the problem was actually in the psp iso, I managed to use a patch to eliminate this slowness that occurs. I'm going to leave the website where I downloaded it here in case anyone else has this problem too, thanks anyway, I'll enjoy the textures!

https://ffhacktics.com/smf/index.php?topic=9523.0

**Comment by blondejefe** [comment]
Simply amazing! Thank you so much, I can't wait to see what future updates hold.

**Comment by isaachermens** [comment]
You mention not wanting to compromise the original sprites. Is it possible to replace character sprites using this method?

Thanks so much for your work!

**Reply by Zodi-ark** [reply]
It's possible but it would take an immense amount of time. The texture replacement format works by pulling up the textures address in memory then the CLUT (aka the palette) and after that the unique data giblits. Its all stored in hashes - 8 for each. The problem comes when the battlefields lighting is randomized which tones the sprites palette. So whenever the lighting conditions change wherever your replacement sprite is you would have to manually find that hash and set it to your new texture, It's not all bad though. In instances of unique events and or boss battles with set lighting its not much of a problem. Furthermore, you could replace sprites + all their variations of palettes that are never used in the game. Meaning you can even go over the sprite limit if you wanted.

I'll throw in some old examples:


https://github.com/Zodi-ark/Final-Fantasy-Tactics-The-War-of-the-Lions-Texture-Pack/assets/113886368/2c180232-7f63-4c86-957c-c9feaf3519ea



https://github.com/Zodi-ark/Final-Fantasy-Tactics-The-War-of-the-Lions-Texture-Pack/assets/113886368/94a5d553-184c-4a03-a1d7-27795000763a

**Reply by isaachermens** [reply]
Those are super cool! Its a bummer that it isn't more feasible :(

**Comment by odevleo** [comment]
Hey man, how's it going? What version of the game are you using? Europe or USA?

**Comment by odevleo** [comment]
After chapter 2 I started having some problems with frame drops again. I'm using the European version.

**Reply by Zodi-ark** [reply]
Make sure you're using the slowdown removal patch. 

https://ffhacktics.com/wiki/Tools#FFTPatcher_Suite

"Open your game with the fftpatcher and just click File -> new psp patch, then PSP -> Patch Wotl ISO -> check the "apply slowdown fix" box."
